ROMFS/px4fmu_common/init.d-posix/airframes/70000_gz_atmos

```diff
@@ -1,10 +1,21 @@
 #!/bin/sh
 #
-# @name 3DoF Spacecraft Model
+# @name KTH-ATMOS
 #
-# @type 2D Freeflyer with 8 thrusters - Planar motion
+# @type Free-Flyer
+# @class Spacecraft
 #
-# @maintainer Pedro Roque <padr@kth.se>
+# @output Motor1 back left thruster, +x thrust
+# @output Motor2 front left thruster, -x thrust
+# @output Motor3 back right thruster, +x thrust
+# @output Motor4 front right thruster, -x thrust
+# @output Motor5 front left thruster, +y thrust
+# @output Motor6 front right thruster, -y thrust
+# @output Motor7 back left thruster, +y thrust
+# @output Motor8 back right thruster, -y thrust
+#
+# @maintainer DISCOWER
```

Comment:
Who is this and how would I find them if I had a question?

Reply:
Ah then i think we could use GitHub.com/DISCOWER . I'll update it in the next days. Thanks for the feedback @hamishwillee !